skyportal/enum_types.py

```diff
@@ -7,14 +7,25 @@
 def force_render_enum_markdown(values):
     return ', '.join(list(map(lambda v: f'`{v}`', values)))
 
+
 ALLOWED_MAGSYSTEMS = tuple(l['name'] for l in _MAGSYSTEMS.get_loaders_metadata())
 ALLOWED_BANDPASSES = tuple(l['name'] for l in _BANDPASSES.get_loaders_metadata())
 THUMBNAIL_TYPES = ('new', 'ref', 'sub', 'sdss', 'dr8', "new_gz", 'ref_gz', 'sub_gz')
+INSTRUMENT_TYPES = ('imager', 'spectrograph', 'imaging spectrograph')
+FOLLOWUP_REQUEST_TYPES = ('imaging', 'spectroscopy')
+FOLLOWUP_PRIORITIES = ('1', '2', '3', '4', '5')
```

Comment:
Why isn't this numerical? And does it have to be enumerated? Does it make sense to have a priority 1.5, which would slot it between 1 and 2?

Reply:
Precedent. Not sure how much granularity is needed for priority, not sure if it makes sense to have a priority 3.2142234142832 assignment. What do you think?

Reply:
Relegating this to a future pr.